src/rgw/rgw_keystone.h

```diff
@@ -255,8 +255,15 @@ class TokenCache {
       cct(g_ceph_context),
       lock("rgw::keystone::TokenCache"),
       max(cct->_conf->rgw_keystone_token_cache_size) {
+    /* revocation logic needs to be smarter, but meanwhile,
+     *  make it optional.
+     * see http://tracker.ceph.com/issues/9493
+     *     http://tracker.ceph.com/issues/19499
+     */
+    if (cct->_conf->rgw_keystone_revocation_interval) {
     /* The thread name has been kept for backward compliance. */
-    revocator.create("rgw_swift_k_rev");
+        revocator.create("rgw_swift_k_rev");
```

Comment:
It would be nice to have the 2-space indentation for this changed line as well as the comment that became a part of the `if` block.